deeplearning4j-core/src/test/java/org/deeplearning4j/nn/layers/recurrent/GravesBidirectionalLSTMTest.java

```diff
@@ -169,21 +177,22 @@ public void testGravesBidirectionalLSTMForwardPassHelper() throws Exception {
         lstm.setInput(input);
 
 
+        LayerNormalization layerNormalization=useLayerNormalization?new LayerNormalizationWholeMinibatch() : new NoNormalization();
         final INDArray fwdPassFalse = LSTMHelpers.activateHelper(lstm, lstm.conf(), new ActivationSigmoid(),
                         lstm.input(),
                         lstm.getParam(GravesBidirectionalLSTMParamInitializer.RECURRENT_WEIGHT_KEY_FORWARDS),
                         lstm.getParam(GravesBidirectionalLSTMParamInitializer.INPUT_WEIGHT_KEY_FORWARDS),
                         lstm.getParam(GravesBidirectionalLSTMParamInitializer.BIAS_KEY_FORWARDS), false, null, null,
                         false, true, GravesBidirectionalLSTMParamInitializer.INPUT_WEIGHT_KEY_FORWARDS,
-                        null).fwdPassOutput;
+                        null,layerNormalization).fwdPassOutput;
 
```

Comment:
Not sure what you mean. A getter on what? It's in a test with no access to the configuration instance, called for each type of layer.